pkg/kubelet/kubelet.go

```diff
@@ -963,6 +965,9 @@ type Kubelet struct {
 	// Container runtime.
 	containerRuntime kubecontainer.Runtime
 
+	// Container runtime service (needed by container runtime)
```

Comment:
we hold a lot of stuff in kubelet, so this is not much worse than others.

that said, add a //TODO to see if we can make this available in the future without holding it.

Reply:
Added TODO.